tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/GraphQLTests.cs

```diff
@@ -185,12 +186,14 @@ public abstract class GraphQLTests : TracingIntegrationTest
 
         private readonly string _testName;
         private readonly string _metadataSchemaVersion;
+        private readonly Regex _timeUnixNanoRegex = new(@"time_unix_nano"":([0-9]{10}[0-9]+)");
 
         protected GraphQLTests(string sampleAppName, ITestOutputHelper output, string testName, string metadataSchemaVersion)
             : base(sampleAppName, output)
         {
             SetServiceVersion(ServiceVersion);
             SetEnvironmentVariable("DD_TRACE_SPAN_ATTRIBUTE_SCHEMA", metadataSchemaVersion);
+            SetEnvironmentVariable("DD_TRACE_GRAPHQL_ERROR_EXTENSIONS", "bool,int,float,test,str,other");
```

Comment:
I don't think any of these show up in the snapshots I'm assuming they would need to be added in some way in the sample apps?

Reply:
Is added to the Span Events in the snapshots for some spans where is relevant.